apps/src/sites/studio/pages/home/teacher_homepage.js

```diff
@@ -19,7 +19,7 @@ function showTeacherHomepage() {
           heading: "Go beyond an Hour of Code",
           buttonText: "Go Beyond",
           description: "Go Beyond an Hour of Code and explore computer science concepts with your students every week. Code.org offers curriculum, lesson plans, high quality professional learning programs, and tons of great tools for all grade levels - and it's free. No experience required - find the next step that's right for your classroom.",
-          link: "to wherever"
+          link: "http://teacherblog.code.org/"
```

Comment:
<img width="783" alt="screen shot 2017-05-12 at 11 15 34 am" src="https://cloud.githubusercontent.com/assets/12300669/26011187/ebf98ca0-3704-11e7-9ef3-efd1934d9007.png">
I'll leave as is for now.